README.md

```diff
@@ -1,24 +1,132 @@
+  <ul>
+    <li>Ruby on Rails</li>
+    <li>React</li>
+  </ul>
+</details>
 
-* Services (job queues, cache servers, search engines, etc.)
+### Key Features <a name="key-features"></a>
 
-* Deployment instructions
+- **Create the Greeting component that will display a greeting**
+- **Create an API endpoint that selects a random greeting from the database table**
+- **Create a store, an action, and a reducer that will connect to your API endpoint to get the random greeting**
 
-* ...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+to install git on your computer. Follow the instruction on the page to do so
+[Click here ](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) and follow the instruction on the page to install git.
+[Click here ](https://www.ruby-lang.org/en/documentation/installation/) and follow the instruction on the page to install ruby.
+[Click here ](https://guides.rubyonrails.org/v5.1/getting_started.html) and follow the instruction on the page to install rails.
+[Click here ](https://www.postgresql.org/download/) and follow the instruction on the page to install postgres.
+
+### Setup
+
+Open your preferred folder in the terminal
+
+Clone this repository to your desired folder:
+Using the commands below:
+
+```sh
+  git clone https://github.com/emarkees/hello-rails-react.git
+```
+
+Open the project on your code editor
+
+Change directory to the project folder
+
+```sh
+  cd hello-rails-react
+```
+
+### Usage
+
+Then, run the following command to start the app
+
+```sh
+    ./bin/dev
+```
+
+## 👥 Author <a name="author"></a>
+
+👤 **Adeyemi Olusola**
+
+- GitHub: [@Adeyemi-arobieke](https://github.com/emarkees)
+- Linkedin: [@Adeyemi](https://linkedin.com/in/adeyemi-olusola)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Add calendar to the app**
+- [ ] **Display greeting based on time, day, and holiday**
+- [ ] **Save greeting message to the database**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+- [Coming Soon]
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/emarkees/hello-rails-react/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project... Give it a Star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank to the Microverse team for the learning opportunity
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](../LICENSE.md) licensed.
```

Comment:
- [ ] [REQUIRED] Good job so far! Please remove the `.md` from the `LICENSE` because it is not working 🙏🏼 

![image](https://github.com/emarkees/hello-rails-react/assets/63915024/e5c9c0ab-d7c1-4f04-9cf0-9e034769a855)

Reply:
done